[DOCS] Editing and Markup Adjustments (#262)
uyuni-project/uyuni-docs: modules/administration/pages/image-management.adoc

{productname} supports two distinct build types: dockerfile, and the Kiwi image system.

The Kiwi build type is used to build system, virtual, and other images.

> The image store for the Kiwi build type is pre-defined as a file system directory at [path]``/srv/www/os-images`` on the server.

Reviewer: this probably needs to be done suite-wide :woman_facepalming:

Author: I can use sed and grep to find the instances of filesystem, and then a script to find and replace them in each file.  Seems like something I should check out an issue with a plan of how I can write something to find and replace the use in text.

Reviewer: Sure, go right ahead.

For an example of a complete Kiwi repository served by git, see https://github.com/SUSE/manager-build-profiles/tree/master/OSImage

[NOTE]
====

> You will need at least 1GB of RAM available for Hosts running OS Images built with Kiwi.

Reviewer: ```suggestion
You will need at least 1{nbsp}GB of RAM available for Hosts running OS Images built with Kiwi.
```

Another inconsistency across the whole suite, I think!

/usr/sbin/mgr-package-rpm-certificate-osimage --ca-cert-full-path /root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT
----

The RPM package with the certificate is stored in a salt-accessible directory such as:
----
/usr/share/susemanager/salt/images/rhn-org-trusted-ssl-cert-osimage-1.0-1.noarch.rpm
----

The RPM package with the certificate is provided in the local build host repository:
----
/var/lib/Kiwi/repo
----

[IMPORTANT]
====

> Specify the RPM Package with the {productname} Certificate in the Build source,
> and make sure your build source Kiwi configuration contains `rhn-org-trusted-ssl-cert-osimage` as a required package in the `bootstrap` section.

Reviewer: ```suggestion
Specify the RPM package with the {productname} SSL certificate in the build source, and make sure your Kiwi configuration contains ``rhn-org-trusted-ssl-cert-osimage`` as a required package in the ``bootstrap`` section.
```

/usr/sbin/mgr-package-rpm-certificate-osimage --ca-cert-full-path /root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT
----

The RPM package with the certificate is stored in a salt-accessible directory such as:
----

> /usr/share/susemanager/salt/images/rhn-org-trusted-ssl-cert-osimage-1.0-1.noarch.rpm.

Reviewer 2: ```suggestion
/usr/share/susemanager/salt/images/rhn-org-trusted-ssl-cert-osimage-1.0-1.noarch.rpm
```

----

The RPM package with the certificate is provided in the local build host repository:
----

> /var/lib/Kiwi/repo.

Reviewer 2: ```suggestion
/var/lib/Kiwi/repo
```

----

[IMPORTANT]

> .Specify the RPM Package with the {productname} Certificate in the Build Source

Reviewer: We don't put a title on admonitions, because we don't have a proper style implemented for them (yet!). I would move this information in to be the first line in the admon, rather than a title.

+

[NOTE]
====

> Associate an activation key with an image profile to ensure the image profile uses the correct software channel, and any packages in the software channel.

Reviewer: ```suggestion
Associate an activation key with an image profile to ensure the image profile uses the correct software channel, and any packages.
```

[NOTE]

> .Relationship Between Activation Keys and Image Profiles

Reviewer: Drive-by: Remove this title too.
